File: Golang/Minimum Number of Operations to Convert Time.go
func convertTime(current string, correct string) int {
    diff := convertToMinutes(correct) - convertToMinutes(current)
    order := [4]int{60,15,5,1}
    ops := 0
    for i := 0 ; i < 4 ; i++ {
        ops+=(diff/order[i])
        diff%=order[i]
    }

    return ops
}

func convertToMinutes(s string) int {
    split := strings.Split(s, ":")
    hours, _ := strconv.Atoi(split[0])
    minutes, _ := strconv.Atoi(split[1])

    return hours * 60 + minutes 
}

File: Python3/Minimum Number of Operations to Convert Time.py
class Solution:
    def convertTime(self, current: str, correct: str) -> int:
        current_time = 60 * int(current[0:2]) + int(current[3:5]) # Current time in minutes
        target_time = 60 * int(correct[0:2]) + int(correct[3:5]) # Target time in minutes
        diff = target_time - current_time # Difference b/w current and target times in minutes
        count = 0 # Required number of operations
		# Use GREEDY APPROACH to calculate number of operations
        for i in [60, 15, 5, 1]:
            count += diff // i # add number of operations needed with i to count
            diff %= i # Diff becomes modulo of diff with i
        return count

#2
class Solution:
    def convertTime(self, current: str, correct: str) -> int:
        current = 60*int(current[0:2]) + int(current[3:])
        correct = 60*int(correct[0:2]) + int(correct[3:])
        diff = correct - current
        return diff//60 + (diff%60) //15 + ((diff%60)%15) //5 + ((diff%60)%15)%5